natbib \defcitealias adds spurious spaces, see bug #628044
fix this by adding some %
---
 texmf-dist/source/latex/natbib/natbib.dtx |    4 ++--
 texmf-dist/tex/latex/natbib/natbib.sty    |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

Index: texlive-base-2012.20120611/texmf-dist/source/latex/natbib/natbib.dtx
===================================================================
--- texlive-base-2012.20120611.orig/texmf-dist/source/latex/natbib/natbib.dtx	2010-12-07 10:39:19.000000000 +0900
+++ texlive-base-2012.20120611/texmf-dist/source/latex/natbib/natbib.dtx	2012-06-25 10:57:23.330134667 +0900
@@ -3992,8 +3992,8 @@
 % indexing can be achieved.
 %    \begin{macrocode}
 \newcommand\defcitealias[2]{%
-   \@ifundefined{al@#1\@extra@b@citeb}{}
-   {\PackageWarning{natbib}{Overwriting existing alias for citation #1}}
+   \@ifundefined{al@#1\@extra@b@citeb}{}%
+   {\PackageWarning{natbib}{Overwriting existing alias for citation #1}}%
    \@namedef{al@#1\@extra@b@citeb}{#2}}
 \DeclareRobustCommand\citetalias{\begingroup
    \NAT@swafalse\let\NAT@ctype\thr@@\NAT@parfalse\NAT@citetp}
Index: texlive-base-2012.20120611/texmf-dist/tex/latex/natbib/natbib.sty
===================================================================
--- texlive-base-2012.20120611.orig/texmf-dist/tex/latex/natbib/natbib.sty	2010-12-07 10:39:19.000000000 +0900
+++ texlive-base-2012.20120611/texmf-dist/tex/latex/natbib/natbib.sty	2012-06-25 10:57:14.030135437 +0900
@@ -740,8 +740,8 @@
 \DeclareRobustCommand\citefullauthor
    {\citeauthor*}
 \newcommand\defcitealias[2]{%
-   \@ifundefined{al@#1\@extra@b@citeb}{}
-   {\PackageWarning{natbib}{Overwriting existing alias for citation #1}}
+   \@ifundefined{al@#1\@extra@b@citeb}{}%
+   {\PackageWarning{natbib}{Overwriting existing alias for citation #1}}%
    \@namedef{al@#1\@extra@b@citeb}{#2}}
 \DeclareRobustCommand\citetalias{\begingroup
    \NAT@swafalse\let\NAT@ctype\thr@@\NAT@parfalse\NAT@citetp}
